app/services/foreman_rh_cloud/cloud_connector.rb

```diff
@@ -0,0 +1,57 @@
+require 'securerandom'
+
+module ForemanRhCloud
+  class CloudConnector
+
+    CLOUD_CONNECTOR_USER = 'cloud_connector_user'.freeze
+    CLOUD_CONNECTOR_TOKEN_NAME = 'Cloud connector access token'.freeze
+
+   def install
+      user = service_user
+      token_value = personal_access_token(user)
+
+      composer = ::JobInvocationComposer.for_feature(
+        'ansible_configure_cloud_connector',
+        [foreman_host.id],
+        {:satellite_user => service_user.login, :satellite_password => token_value}
+      )
+      composer.trigger!
+      composer.job_invocation
```

Comment:
From UI point of view we will want to show that the job is still running. For that, we will need to fetch the correct job invocation from the UI, where we don't have the ID. Is there a way we can identify the correct invocation?

Reply:
`compose.job_invocation` should be the job invocation that was created, so it is just a matter of serializing it into json in the controller.

Reply:
@adamruzicka, in the current browser session it will be there, sure. What about refresh? I need a way to find the invocation with fresh browser (redux store)

Reply:
That should be a responsibility of that part of the frontend code which calls this api, no?